src/DotNest.Core.SDK.Web/Program.cs

```diff
@@ -17,8 +20,17 @@
 
     if (builder.Environment.IsDevelopment())
     {
-        orchardCoreBuilder.AddSetupFeatures("OrchardCore.AutoSetup");
+        orchardCoreBuilder
+                .AddSetupFeatures("OrchardCore.AutoSetup")
+                .HideRecipesByTagsFromSetup("test", "HideFromSetupScreen");
```

Comment:
There is one plus indentation.
 
```suggestion
        orchardCoreBuilder
            .AddSetupFeatures("OrchardCore.AutoSetup")
            .HideRecipesByTagsFromSetup("test", "HideFromSetupScreen");
```